walkthroughs/howto-ingress-enhancements/infrastructure/ecs-service-with-alb.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```

Comment:
Do we still need these ALB files?

Reply:
Deleted, sorry it got added back during the transfer.